pkg/accounting/tally.go

```diff
@@ -0,0 +1,170 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package accounting
+
+import (
+	"context"
+	"time"
+
+	"github.com/gogo/protobuf/proto"
+	"go.uber.org/zap"
+
+	"storj.io/storj/pkg/dht"
+	"storj.io/storj/pkg/kademlia"
+	"storj.io/storj/pkg/node"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/pointerdb"
+	"storj.io/storj/pkg/provider"
+	"storj.io/storj/pkg/utils"
+	"storj.io/storj/storage"
+)
+
+// Tally is the interface for the data accounting tally
```

Comment:
Explain the idea without using word `tally`. e.g. `// Tally is a service for adding up storage node data usage.`